sopel/modules/version.py

```diff
@@ -33,22 +33,43 @@ def git_info():
 
 
 @plugin.command('version')
+@plugin.example('.version')
+@plugin.example('.version url')
```

Comment:
Like the last time I commented on a PR of yours that added examples, these won't work as intended. Without using `user_help=True`, only the "first" (closest to the `def`) example will be shown in `.help` output.

In this case, since the example isn't being (ab)used to generate a test case, might as well write it like a usage string:

```suggestion
@plugin.example('.version [plugin_name]')
```

Reply:
The ordering was intended since `.version sopel` works too, but ack on brackets.